app/_includes/nav.html

```diff
@@ -496,6 +498,19 @@
                       Kong Enterprise</a
                     >
                   </li>
+                  <li
+                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-12489 navbar-item"
+                  >
+                    <a
+                      target="_blank"
+                      href="https://docs.konghqdev.wpengine.com/mesh/?itm_source=website&itm_medium=nav"
```

Comment:
This looks like an internal link @adamjkuhn

Reply:
Yikes thanks @lena-larionova - updated!